code/modules/mob/say_vr.dm

```diff
@@ -108,7 +108,9 @@
 
 		for(var/vismob in vis_mobs)
 			if(istype(vismob, /mob/observer))
-				continue
+				var/mob/observer/O = vismob
+				if(O.client && !check_rights(R_ADMIN, FALSE, O.client))
```

Comment:
@silicons or any other maint best look at this as I couldn't proccall a mob without being an admin to check if it works right